<launch>
    <node name="test_controller" pkg="robot2017" type="test_controller" output="screen" respawn="true" respawn_delay="10">
    </node>
</launch>
